ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -337,3 +337,42 @@ def test_jax_numpy_sort_complex(
         a=x[0],
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="jax.numpy.where",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("numeric")),
+    test_with_out=st.just(False),
+    inplace=st.just(False),
+)
+def test_jax_numpy_where(
+    *,
+    dtype_and_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
```

Comment:
all of these can be removed,
you can instead just add argument `test_flags`

Reply:
Thanks, I have removed it